src/components/v2/chartDeploymentHistory/DockerListModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { ReactComponent as Close } from '../assets/icons/ic-close.svg'
+import docker from '../../../assets/icons/misc/docker.svg'
+
+export default function DockerListModal({ closeTab, dockerList }) {
+    return (
+        <div className="docker-wrapper" onClick={(e) => e.stopPropagation()}>
+            <div className="docker-header flex-align-center fs-16 fw-6 pl-20 pr-20 pt-17 pb-17">
+                <span className='cn-9'>Deployed images</span>
+                <button type="button" className="transparent flex" onClick={() => closeTab()}>
```

Comment:
Always try to avoid inline functions i.e. onClick={() => closeTab()} to onClick={closeTab}

Reply:
Done